torch_frame/data/multi_nested_tensor.py

```diff
@@ -224,6 +231,29 @@ def stack(xs: List['MultiNestedTensor'],
         else:
             raise NotImplementedError
 
+    @staticmethod
+    def cat(xs: List['MultiNestedTensor'],
```

Comment:
I think in PyTorch `Tuple` of `Tensors` are also supported. We should implement this function with the same arguments as PyTorch `cat`.